BraveRewardsUI/Settings/Auto-Contribute/Exclusion List/AutoContributeExclusionListController.swift

```diff
@@ -78,7 +78,7 @@ final class AutoContributeExclusionListController: UIViewController {
   }
   
   @objc private func tappedRestoreAll(_ sender: UIBarButtonItem) {
-    let numberOfExcludedSites = state.ledger.numberOfExcludedPublishers
+    let numberOfExcludedSites = publishers.count
```

Comment:
not sure about naming but shouldn't this be some kind of excluded publishers count?

Reply:
We are in a controller which only shows excluded publishers, so any publisher in here is already excluded